src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/mediapicker/mediapicker.controller.js

```diff
@@ -557,6 +557,11 @@ angular.module("umbraco")
 
                     var allowedTypes = dialogOptions.filter ? dialogOptions.filter.split(",") : null;
 
+                    if ($scope.model.sortBy === "updateDate") {
+                      data = sortByUpdateDate(data, $scope.model.sortType);
+                    } else {
+                      data = sortByUpdateName(data, $scope.model.sortType);
+                    }
```

Comment:
It could probably just be `sortByProperty(data, property, direction)`

Reply:
resolved